fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1651,7 +1652,7 @@ void SimulationConfig::setTss(const TestConfig& testConfig) {
 		std::string confStr = format("tss_count:=%d tss_storage_engine:=%d", tssCount, db.storageServerStoreType);
 		set_config(confStr);
 		double tssRandom = deterministicRandom()->random01();
-		if (tssRandom > 0.5) {
+		if (tssRandom > 0.5 || !faultInjectionActivated) {
```

Comment:
This should be moved to line 1651 instead, to skip `set_config`.

Reply:
Thanks, Updated.